index.js

```diff
@@ -1,18 +1,33 @@
 /***** Deliverable 1 *****/
+
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "blue"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+document.getElementsByTagName('img')[0].src = traveler.photo
+document.getElementsByTagName('img')[0].alt = "Raffy pic"
+document.getElementsByTagName('h2')[0].innerText = traveler.name
+document.getElementsByTagName('em')[0].innerText = traveler.nickname
+console.log('This is what the traveler object looks like: ', traveler)
 
```

Comment:
Nice! A way to optimize this would be to store the value of `document.getElementsByTagName('img')[0]` in a variable, so that way you can reference the variable on the updates instead of needing JS to search for and find it again.

```js
const img = document.getElementsByTagName('img')[0].src // traversing the DOM _once_ to find the element & storing a reference to it
img.src = traveler.photo
img.alt = "Raffy pic"
```